src/Messages/OWSSignalServiceProtos.pb.m

```diff
@@ -5865,6 +5916,26 @@ - (OWSSignalServiceProtosAttachmentPointerBuilder*) clearFileName {
   resultAttachmentPointer.fileName = @"";
   return self;
 }
+- (BOOL)hasFlags
```

Comment:
For some reason editing generated code, even if it's just clang-format, gives me the howling fantods. But I'm willing to accept that this is a reasonable thing to do.